style.css

```diff
@@ -0,0 +1,414 @@
+  font-size: 20px;
+  line-height: 24px;
+  height: 25px;
+  width: 31px;
+}
+
+.header__language-ru:: after {
+  content: "";
+  color: #fff;
+  position: absolute;
+}
+
+.header__language-ru: hover {
+  transform: scale(1.3);
+}
+
+.header__menu {
+  display: flex;
+  margin-left: auto;
+  position: relative;
+  align-self: flex-end;
+  width: 70px;
+  height: 70px;
+  background: #f5f5f3;
+  cursor: pointer;
+}
+
+.header__menu: hover {
+  background: #696969;
+}
+
+.inst {
+  background: #fff url('img/inst_cir.png') center no-repeat;
+  width: 43px;
+  height: 43px;
+  border-radius: 50%;
+  left: 52px;
+  top: 816px;
+  display: inline-block;
+}
+
+.inst:hover {
+  background: linear-gradient(to bottom, #833ab4, #c13584, #f77737, #ffdc80);
+}
+
+.vk {
+  background: #fff url('img/vk.png') center no-repeat;
+  width: 43px;
+  height: 43px;
+  border-radius: 50%;
+  left: 107px;
+  top: 816px;
+  display: inline-block;
+}
+
+.vk:hover {
+  background: #5b88bd url('img/vk.png') center no-repeat;
+}
+
+.fb {
+  background: #fff url('img/fb.png') center no-repeat;
+  width: 43px;
+  height: 43px;
+  border-radius: 50%;
+  left: 162px;
+  top: 816px;
+  display: inline-block;
+}
+
+.fb:hover {
+  background: #4267b2 url('img/fb.png') center no-repeat;
+}
+
+.block {
+  position: absolute;
+  top: 168px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  max-width: 1500px;
+}
+
+.menu__item {
+  list-style-type: none;
+  padding: 0;
+  margin: auto;
+}
+
+#menu__toggle {
+  display: none;
+}
+
+.header__burger {
+  right: 0;
+  margin-left: auto;
+  display: flex;
+  position: absolute;
+  width: 92px;
+  height: 92px;
+  background-color: #ededed;
+}
+
+.header__burger > span,
+.header__burger > span::after {
+  display: block;
+  position: absolute;
+  width: 34.7px;
+  height: 3.48px;
+  margin-left: 28px;
+  top: 39px;
+  background-color: #000;
+}
+
+.header__burger > span::after {
+  content: '';
+  margin-left: 0;
+  top: 6px;
+}
+
+  @media (min-width: 992px) {
+    .header__menu {
+      display: flex;
+      min-height: 92px;
+      min-width: 92px;
+    }
+    .header__menu:hover {
+        background: #696969;
+      }
+  }
+
+  .menu__link {
+    color: navy;
+    font-size: 25px;
+  }
+
+  .icon__line:nth-child(1) {
+    top: 0;
+  }
+
+  .icon__line:nth-child(2),
+  .icon__line:nth-child(3) {
+    top: 13px;
+  }
+
+  .icon__line:nth-child(4) {
+    top: 26px;
+  }
+
+  .menu__line:first-of-type {
+    margin-bottom: 5px;
+  }
+
+  @media (max-width: 900px) {
+    .header__menu {
+      width: 100%;
+      display: none;
+    }
+
+    .header__menu:hover {
+        background: #696969;
+      }
+
+    .header__burger {
+      width: 69px;
+      height: 69px;
+    }
+
+    .header__burger > span {
+      top: 29px;
+      margin-left: 21px;
+    }
+
+    .header__burger > span::after {
+      content: '';
+      margin-left: 0;
+    }
+  }
+
+  .new-collection {
+    position: relative;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5986341)